explore/src/main/scala/explore/tabs/TargetTabContents.scala

```diff
@@ -130,6 +141,30 @@ object TargetTabContents extends TwoPanels:
     )
   )
 
+  private val singleLayoutMedium: Layout = Layout(
+    List(
+      LayoutItem(
+        i = ObsTabTilesIds.TargetSummaryId.id.value,
+        x = 0,
+        y = 0,
+        w = DefaultWidth.value,
+        h = 0, // This doesn't matter, we are forcing 100%.
```

Comment:
How are we doing the 100% forcing?

Reply:
> How are we doing the 100% forcing?

Via `ExploreStyles.SingleTileMaximized`, which I see you mentioned in another comment. It justs sets `height: 100% !important` (same for `width`) in the `TileController` and the `Tile`.